apps/molgenis-components/src/components/tables/RefTable.vue

```diff
@@ -94,6 +94,9 @@ let filteredResults = computed(() => getFilteredResults(reference.value));
 let canCollapse = computed(() => Object.keys(filteredResults.value).length > 3);
 
 let collapsed = ref(startsCollapsed.value && canCollapse.value);
+let primaryKey = computed(() =>
+  getPrimaryKey(reference.value, reference.value.metadata)
+);
```

Comment:
minor thingy, let's put computeds together if possible, in order to keep to files more clean.